lib/replicache.types.ts

```diff
@@ -0,0 +1,50 @@
+import type { MutationV1 } from 'replicache'
+import { z } from 'zod'
```

Comment:
Yeah, I think it's fine to introduce Zod as a dependency here. Most people at this point have some exposure to Zod (or something like it), so this seems fine.